### PR TITLE
Remove "e" from FAQ entry

### LINKED ISSUE #790: FAQ: Spurious character "e" in title of Cause 2001
## Where to find the issue
https://www.coronawarn.app/de/faq/#cause2001

## Describe the issue
The title "URSACHE: 2001, trying to **e** establish a secure connection to the server" contains the extra letter "e".

## Suggested change
Change the title to
"URSACHE: 2001, trying to establish a secure connection to the server"

See also comment https://github.com/corona-warn-app/cwa-website/pull/783#issuecomment-761549187 in related PR.

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR removes the spurious "e" in the `cause2001` FAQ entry.

Closes #790

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -35,17 +35,17 @@
...
-                        "anchor": "cause2001",
-                        "active": true,
-                        "textblock": [
+                    {
+                        "title": "URSACHE: 2001, trying to establish a secure connection to the server",
+                        "anchor": "cause2001",
+                        "active": true,
+                        "textblock": [
                             "Das Sicherheitszertifikat <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b> ist möglicherweise auf Ihrem Gerät deaktiviert. Dies ist eine bekannte Ursache, um folgende Benachrichtigung auszulösen:",
-                            "<b>'URSACHE: 2001 Etwas ist schief gelaufen. An error occurred while trying to establish a secure connection to the server.'</b>",
```
Comment (Ein-Tim) [inline src/data/faq_de.json:44]: Again, did not change anything here, dont know why this is happening.

Comment (Ein-Tim) [inline src/data/faq_de.json:44]: Again, did not change anything here, don't know why this is happening

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 thanks

File: src/data/faq_de.json (hunk 1 above)
Comment (MikeMcC399) [reply src/data/faq_de.json:44]: @Ein-Tim 
> Again, did not change anything here, don't know why this is happening

Your editor is changing whitespace. If you go to "Files changed", click on the gear-wheel, then select "Hide whitespace changes" , Apply and reload, you will see that the unexpected changes disappear from the view.

**Comment by MikeMcC399** [comment]
@Ein-Tim 
The whitespace issue is caused by ^M = carriage return.
GitHub does not show this, but if you use git instead at a bash prompt and compare the two commits then you can see the difference (in colour if you do it yourself).

---
```
$ git diff 114bcf9 ce79769
diff --git a/src/data/faq_de.json b/src/data/faq_de.json
index 4be6ffb..45f1f79 100644
--- a/src/data/faq_de.json
+++ b/src/data/faq_de.json
@@ -35,17 +35,17 @@
                 "title": "Häufige Probleme",
                 "id": "most_asked",
                 "accordion": [
-                    {
-                        "title": "URSACHE: 2001, trying to establish a secure connection to the server",
-                        "anchor": "cause2001",
-                        "active": true,
-                        "textblock": [
+                    {^M
+                        "title": "URSACHE: 2001, trying to e establish a secure connection to the server",^M
+                        "anchor": "cause2001",^M
+                        "active": true,^M
+                        "textblock": [^M
                             "Das Sicherheitszertifikat <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b> ist möglicherweise auf Ihrem Gerät deaktiviert. Dies ist eine bekannte Ursache, um folgende Benachrichtigung auszulösen:",
-                            "<b>'URSACHE: 2001 Etwas ist schief gelaufen. An error occurred while trying to establish a secure connection to the server.'</b>",
-                            "Führen Sie die folgenden Schritte aus, um das Sicherheitszertifikat zu aktivieren (Der genaue Pfad hängt vom Gerät ab).",
-                            "<ul><li>Einstellungen > Biometrische Daten und Sicherheit > Andere Sicherheitseinstellungen > Sicherheitszertifikate anzeigen, SYSTEM</li><li>Scrollen Sie nach unten zu <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b></li><li>Aktivieren Sie das Zertifikat mit dem Schieberegler</li></ul>",
-                            "Sollten diese Prozedur das Problem nicht beheben, schreiben SIe bitte Ihren Bericht hier: <a href='https://github.com/corona-warn-app/cwa-app-android/issues/968' target='_blank' rel='noopener'>cwa-app-android/issues/968</a>."
-                        ]
+                            "<b>'URSACHE: 2001 Etwas ist schief gelaufen. An error occurred while trying to establish a secure connection to the server.'</b>",^M
+                            "Führen Sie die folgenden Schritte aus, um das Sicherheitszertifikat zu aktivieren (Der genaue Pfad hängt vom Gerät ab).",^M
+                            "<ul><li>Einstellungen > Biometrische Daten und Sicherheit > Andere Sicherheitseinstellungen > Sicherheitszertifikate anzeigen, SYSTEM</li><li>Scrollen Sie nach unten zu <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b></li><li>Aktivieren Sie das Zertifikat mit dem Schieberegler</li></ul>",^M
+                            "Sollten diese Prozedur das Problem nicht beheben, schreiben SIe bitte Ihren Bericht hier: <a href='https://github.com/corona-warn-app/cwa-app-android/issues/968' target='_blank' rel='noopener'>cwa-app-android/issues/968</a>."^M
+                        ]^M
                     },
                     {
                         "title": "Firewall/Router Konfiguration: URSACHE: 4000, error during web request, HTTP status 901",

```
---

Now repeat the command with -b then the difference is much shorter

```
$ git diff 114bcf9 ce79769 -b
diff --git a/src/data/faq_de.json b/src/data/faq_de.json
index 4be6ffb..45f1f79 100644
--- a/src/data/faq_de.json
+++ b/src/data/faq_de.json
@@ -36,7 +36,7 @@
                 "id": "most_asked",
                 "accordion": [
                     {
-                        "title": "URSACHE: 2001, trying to establish a secure connection to the server",
+                        "title": "URSACHE: 2001, trying to e establish a secure connection to the server",^M
                         "anchor": "cause2001",
                         "active": true,
                         "textblock": [

```

I just noticed another typo in the German text, so I'm going to try to correct that and see what happens. I am using Atom as editor, the one recommended by GitHub.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Thank's, I did not know this. I will try Atom now 👍

**Comment by MikeMcC399** [comment]
@Ein-Tim 
I didn't get any extra whitespace in https://github.com/corona-warn-app/cwa-website/pull/792 . It just added a newline at the end of the file.

If you use Atom watch out for the settings in the "whitespace" core package.

**Pull request merged**